README.md

```diff
@@ -16,23 +16,23 @@ To install `package.json` dependencies in a way where the native
 modules are built with `electron`, run:
 
 ```bash
-$ ./scripts/install.sh
+./scripts/install.sh
 ```
 
 Then, you want to make sure `app/dist` is populated. I recommend
 running `webpack` with `--watch` so that any changes you make
 to the app are detected.
 
 ```bash
-$ cd app/
-$ npm install
-$ webpack --watch
+cd app/
+npm install
+webpack --watch
```

Comment:
`npm run dev` instead of `webpack --watch` would be better :)
`webpack --watch` won't work if user doesn't have a global installed webpack.